examples/ember-cli/todomvc/app/templates/application.hbs

```diff
@@ -0,0 +1,29 @@
+<section id="todoapp">
+  <header id="header">
+    <h1>todos</h1>
+    <input type="text" id="new-todo" onkeydown={{action "createTodo"}} placeholder="What needs to be done?" autofocus>
+  </header>
+    {{outlet}}
+    {{#if model.length}}
+      <footer id="footer">
+        <span id="todo-count"><strong>{{remaining.length}}</strong> {{pluralize 'item' remaining.length}} left</span>
```

Comment:
I personally use double quotes in html and single quotes inside braces, but it's just a habit, I don't have strong feelings either way.